packages/design-system/src/components/Form/Row/Row.tsx

```diff
@@ -2,20 +2,29 @@ import React from 'react';
 import { isElement } from 'react-is';
 
 import styles from './Row.module.scss';
+import classNames from 'classnames';
 
 type RowProps = React.HTMLAttributes<HTMLDivElement> & {
 	disabled?: boolean;
 	readOnly?: boolean;
+	isStretched?: boolean;
```

Comment:
Got the whole story from @mhuet but rows are by default meant to be "inlining" stuff.